addOns/help/src/main/javahelp/contents/start/proxies.html

```diff
@@ -9,7 +9,10 @@
 <BODY>
 <H1>Configuring Proxies</H1>
 <p>
-You will need to configure your browser to use ZAP as a proxy.<br>
+The best way to use a browser with ZAP is to launch it from ZAP. 
+It will then be automatically configured to proxy through ZAP and to ignore certificate warnings.
+<p>
+If for any reason you are unable or unwilling to be that then you will need to configure your browser to use ZAP as a proxy.<br>
```

Comment:
be that ➡️ do that ..... as a proxy, and add ZAP's root CA certificate.

Reply:
fixed